lib/pbench/test/unit/server/query_apis/test_controllers_list.py

```diff
@@ -41,32 +57,37 @@ def test_missing_json_object(self, client, server_config):
             {"end": "2020"},
             {"user": "x", "start": "2020"},
             {"user": "x", "end": "2020"},
-            {"start": "2020", "end": "2020"},
+            {"some_additional_key": "test"},
         ),
     )
-    def test_missing_keys(self, client, server_config, keys, user_ok):
+    def test_missing_keys(self, client, server_config, keys, user_ok, pbench_token):
         """
         test_missing_keys Test behavior when JSON payload does not contain
         all required keys.
 
-        Note that "user", "start", and "end" are all required;
+        Note that "start", and "end" are required where as "user" is not mandatory;
```

Comment:
"whereas" is one word. I would have overlooked this, but I suspect Webb would point it out if I don't first. ;-)

Reply:
I caught it, got distracted, and then failed to get back to it.  😀